### ISSUE TITLE
Dependencies Issue 

### ISSUE THREAD

**Issue by srathi1** [direct-code-execution/ns-3-dce#70]
Hi 
I am trying to install NS3-DCE but ending up with missing dependencies "libc" and "pygoocanvas". I already installed these dependencies but it still showing missing dependencies. 
![dce-error](https://user-images.githubusercontent.com/37901338/38087016-69f457cc-330b-11e8-993e-80da76d0c0fb.png)

Moreover i am wondering how to run scripts after adding into "myscripts" folder. 

Thanks

**Comment by teto** [comment]
have you tried running with -vvv as is suggested to see what the actual problem is ?

**Comment by srathi1** [comment]
Ya i tried with -v option and here is the snapshot for that.

![dce-error2](https://user-images.githubusercontent.com/37901338/38232471-06c86bb0-36cc-11e8-9e68-2e38b122daf0.png)

**Comment by teto** [comment]
Have you installed pygoo-canvas for the same version of python ? 2 vs 3
mostly.
I am not too familiar with bake but the config for libc is here
http://code.nsnam.org/bake/file/c502b48053dc/bakeconf.xml#l1087. check that
the .so file exists else change bakeconf.xml for your system. What
distribution are you on ?

Note: Please copy/paste the text rather than using screenshots

2018-04-03 15:17 GMT+09:00 srathi1 <notifications@github.com>:

**Comment by srathi1** [comment]
Yes i have installed pygoo-canvas but i am not sure about version of it and when i checked with commands - bake show & bake check everything was ok. Can you please check with the file "DCE-error.txt" which i uploaded in my last comment , just above the snapshot. 

I am using Ubuntu 16.04

Thanks

**Comment by teto** [comment]
Looking at your log it seems like the install is successful. You are missing some functionalities because you don't have the required dependencies but that's all`'install' finished successfully (0.402s)`

For instance you lack openssl-dev
 ```
ccn_client.c
ccn_client.c:36:25: fatal error: openssl/evp.h: No such file or directory
compilation terminated.
```
and probably LLVM
 ```
 >> Building castxml - mkdir: cannot create directory ‘/home/Multispeak_DCE/dce/source/castxml/objdir’: File exists
CMake Error at CMakeLists.txt:95 (find_package):
  By not providing "FindLLVM.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "LLVM", but
  CMake did not find one.

  Could not find a package configuration file provided by "LLVM" with any of
  the following names:

    LLVMConfig.cmake
    llvm-config.cmake

  Add the installation prefix of "LLVM" to CMAKE_PREFIX_PATH or set
  "LLVM_DIR" to a directory containing one of the above files.  If "LLVM"
  provides a separate development package or SDK, be sure it has been
  installed.

```

 but do you really need those ?
